icubam/www/handlers/home.py

```diff
@@ -40,6 +50,15 @@ def initialize(self, config, db_factory):
     super().initialize(config, db_factory)
     self.regions = None
 
+  def get_disclaimer_url(self):
+    """To show a disclaimer link if defined in configuration."""
+    if self.config.server.has_key('disclaimer'):
+      return "<a href='{}disclaimer'>disclaimer</a>".format(
+        self.config.server.base_url
```

Comment:
```suggestion
      return "<a href='/disclaimer'>disclaimer</a>"
```

because it otherwise it would not allow accessing this URL for the dev server from another location than localhost.